main.py

```diff
@@ -45,159 +41,180 @@
     )
-    post_dataframe_to_google_sheets(
-        df=criteria_6_summary_df,
-        workbook_key=CDC_CRITERIA_6_GOOGLE_WORKBOOK_KEY,
-        tab_name=STATE_SUMMARY_TAB_NAME,
-        credentials=credentials,
-    )
+    if post_to_google_sheets:
+        post_dataframe_to_google_sheets(
+            df=criteria_6_summary_df,
+            workbook_key=CDC_CRITERIA_6_GOOGLE_WORKBOOK_KEY,
+            tab_name=STATE_SUMMARY_TAB_NAME,
+            credentials=credentials,
+        )
 
-    # It appears we're uploading so much data we're hitting a quota of some kind. So pause before this last spreadsheet,
-    # which is enormous.
-    sleep_and_log()
+        sleep_and_log()
 
     # Merge all the summary data frames so that we can create a single summary sheet.
     combined_df = functools.reduce(
         # Use an inner join so that you'll only get entities that are represented in all criteria.
         # E.g., you'll not include Guam, NYC, etc which are reported separately in some of the sources.
-        lambda left, right: pd.merge(left, right, on=[STATE_FIELD], how="inner"),
+        lambda left, right: pd.merge(
+            left, right, on=[STATE_FIELD], how="inner", suffixes=["", "_y"]
+        ),
         [
             criteria_1_summary_df,
             criteria_2_summary_df,
-            criteria_3_summary_df,
+            # TODO(lbrown): Un-comment this when we find a path forward for CDC bed data.
+            # criteria_3_summary_df,
             criteria_5_summary_df,
             criteria_6_summary_df,
         ],
     )
 
-    post_dataframe_to_google_sheets(
-        df=combined_df.loc[:, CRITERIA_COMBINED_SUMMARY_COLUMNS],
-        workbook_key=CDC_CRITERIA_SUMMARY_GOOGLE_WORKBOOK_KEY,
-        tab_name=STATE_SUMMARY_TAB_NAME,
-        credentials=credentials,
-    )
+    if post_to_google_sheets:
+        post_dataframe_to_google_sheets(
+            df=combined_df.loc[:, CRITERIA_COMBINED_SUMMARY_COLUMNS],
+            workbook_key=CDC_CRITERIA_SUMMARY_GOOGLE_WORKBOOK_KEY,
+            tab_name=STATE_SUMMARY_TAB_NAME,
+            credentials=credentials,
+        )
 
-    sleep_and_log()
+        sleep_and_log()
 
-    # Upload data for all states.
-    post_dataframe_to_google_sheets(
-        df=transformed_covidtracking_df,
-        workbook_key=CDC_GUIDANCE_GOOGLE_WORKBOOK_KEY,
-        tab_name=ALL_STATE_DATA_TAB_NAME,
-        credentials=credentials,
-    )
+        # Upload data for all states.
+        post_dataframe_to_google_sheets(
+            df=transformed_covidtracking_df,
+            workbook_key=CDC_GUIDANCE_GOOGLE_WORKBOOK_KEY,
+            tab_name=ALL_STATE_DATA_TAB_NAME,
+            credentials=credentials,
+        )
 
 
 if __name__ == "__main__":
-    extract_transform_and_load_covid_data()
+    # Note: for faster debugging during development, you can set `post_to_google_sheets` to `False`.
+    extract_transform_and_load_covid_data(post_to_google_sheets=True)
```

Comment:
I'm not sure what y'all think of this new parameter; it was helpful for me during development.

Reply:
I love it! See my previous comment about potentially writing to local CSVs when this flag is toggled off.